src/main/java/school/hei/haapi/endpoint/rest/controller/StudentController.java

```diff
@@ -41,19 +43,21 @@ public Student getStudentById(
   @GetMapping("/students")
   public List<Student> getStudents(
       @RequestParam PageFromOne page, @RequestParam("page_size") BoundedPageSize pageSize) {
-    return userService
-        .getByRole(User.Role.STUDENT, page, pageSize).stream()
-        .map(userMapper::toRestStudent)
-        .collect(toUnmodifiableList());
+    List<Student> students =  userService.getByRole(User.Role.STUDENT, page, pageSize).stream()
+            .map(userMapper::toRestStudent)
+            .collect(toUnmodifiableList());
+    return students;
   }
 
   @PutMapping("/students")
-  public List<Student> saveAll(@RequestBody List<Student> toSave) {
+  public List<Student> saveAll(@RequestBody List<Student> toWrite) {
+    if (toWrite.size() > EventConf.MAX_EVENT_REQUEST_ENTRY) {
+      throw new BadRequestException(
+          String.format(
+              "Students to create or update must be ≤ %d", EventConf.MAX_EVENT_REQUEST_ENTRY));
```

Comment:
Just `BadRequestException("string" + MAX_PUT_EVENT_ENTRIES)` should be enough. The use of `String.format` is just cumbersome without adding any real value.